setup.py

```diff
@@ -2,17 +2,35 @@
 from setuptools import setup, find_packages
 
 
+VERSION = '0.3.0'
+
+
 setup(
   name='AeroGear build cli for Digger',
-  version='0.3.0',
+  version=VERSION,
   description='AeroGear build cli for Digger',
   author='AeroGear',
+  author_email = 'aerogear-dev@lists.jboss.org',
```

Comment:
do we need to 'sign up' somewhere ? 

If yes, than let's use aerogear AT redhat DOT com